platforms/enterprise/enterprise/src/main/java/org/gradle/internal/enterprise/impl/GradleEnterpriseAutoAppliedPluginRegistry.java

```diff
@@ -50,14 +50,19 @@ public PluginRequests getAutoAppliedPlugins(Settings target) {
         if (((StartParameterInternal) target.getStartParameter()).isUseEmptySettings() || !shouldApplyDevelocityPlugin(target)) {
             return PluginRequests.EMPTY;
         } else {
+            // We are going with an auto-application request, let's configure the URL
+            // TODO Remove this once the default applied version supports DefaultGradleEnterprisePluginConfig.getDevelocityUrl()
+            target.getPluginManager().withPlugin(AutoAppliedDevelocityPlugin.ID.getId(), new DevelocityAutoAppliedPluginConfigurationAction(target));
```

Comment:
Like I said above, use the system property. If you do this here you would also configure the plugin when the build applies and configures it already.